tests/foreman/virtwho/test_api_virtwho.py

```diff
@@ -107,3 +111,243 @@ def test_positive_deploy_configure_id(self):
         vhd.delete()
         self.assertEqual(entities.VirtWhoConfig().search(
             query={'search': 'name={}'.format(name)}), [])
+
+    @tier1
+    def test_positive_deploy_configure_by_script(self):
+        """ Verify "GET /foreman_virt_who_configure/api/
+
+        v2/configs/:id/deploy_script"
+
+        :id: bb673b27-c258-4517-8fb9-436a4b51ba9d
+
+        :expectedresults: Config can be created and deployed
+
+        :CaseLevel: Integration
+
+        :CaseImportance: High
+        """
+        name = gen_string('alpha')
+        args = self._make_virtwho_configure()
+        args.update({'name': name})
+        vhd = entities.VirtWhoConfig(**args).create()
+        self.assertEqual(vhd.status, 'unknown')
+        script = vhd.deploy_script()
+        hypervisor_name, guest_name = deploy_configure_by_script(
+            script['virt_who_config_script'], debug=True)
+        status = entities.VirtWhoConfig().search(
+            query={'search': 'name={}'.format(name)}
+        )[0].status
+        self.assertEqual(status, 'ok')
```

Comment:
```suggestion
self.assertEqual(entities.VirtWhoConfig().
                 search(query={'search': 'name={}'.format(name)})[0].status),
                 'ok')
```
Could you please try as above ?

Reply:
Yeah, it can work, the reason why I split it is that I thought the line was too long, but if you think this may rather perfect, I can optimize it with the new commit. : )